src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs.cljs

```diff
@@ -99,9 +101,12 @@
                                             (react/call :load-method-configs this)
                                             (swap! state update-in [:method-confs]
                                               assoc (:selected-index @state) new-conf)))}]
-         (:method-confs-loaded? @state) [WorkspaceMethodsConfigurationsList {:workspace (:workspace props)
-                                                                             :method-confs (:method-confs @state)
-                                                                             :parent-state state}]
+         (:method-confs-loaded? @state) [WorkspaceMethodsConfigurationsList
+                                          {:onImport
+                                             (fn [] (react/call :load-method-configs this))
```

Comment:
This is right for the real data, but it's going to clobber mock data.  I think this callback function should take the imported config as a parameter, and then in the case of mock data, this code can just `conj` it onto the mock configs list.